### PR TITLE
Fix lineage integration doc

### PR DESCRIPTION
**Pull request by paolaperaza**

### CODE REVIEW

File: astro/set-up-data-lineage.md [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
+sidebar_label: 'Integrate lineage'
+title: "Integrate lineage with external systems"
 id: set-up-data-lineage
 description: Configure your external systems to emit OpenLineage data to Astro with Apache Airflow.
 toc_min_heading_level: 2
 toc_max_heading_level: 2
 ---
 
-[Data lineage](https://en.wikipedia.org/wiki/Data_lineage) is the concept of tracking data from its origin to wherever it is consumed downstream as it flows through a data pipeline. This includes connections between datasets and tables in a database as well as rich metadata about the tasks that create and transform data. You can observe data lineage to trace the history of a dataset, troubleshoot run failures, manage personally identifiable information (PII), and ensure compliance with data regulations.
+[Data lineage](https://en.wikipedia.org/wiki/Data_lineage) is the concept of tracking data from its origin to wherever it is consumed downstream as it flows through a data pipeline. This includes connections between datasets and tables in a database as well as rich metadata about the tasks that create and transform data. You can observe data lineage to:
```
Comment (kwcanuck) [inline astro/set-up-data-lineage.md:10]:
```suggestion
[Data lineage](https://en.wikipedia.org/wiki/Data_lineage) is the concept of tracking data from its origin to wherever it is consumed downstream as it flows through a data pipeline. This includes connections between datasets and tables in a database as well as rich metadata about the tasks that create and transform data. This guide describes what lineage data is automatically extracted on Astro and how to integrate external applications, including Databricks and dbt, that require additional configuration. To view data lineage on Astro, see [View data lineage](data-lineage.md).

Data lineage data can help you:
```

File: astro/set-up-data-lineage.md [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
 
-To store lineage data that is emitted from any system, a lineage backend is required. Astro Deployments use [OpenLineage](https://openlineage.io/integration/apache-airflow/) (`openlineage-airflow`) as a backend for gathering lineage data. The OpenLineage Airflow library is installed on [Astro Runtime](runtime-image-architecture.md) by default.
+- Trace the history of a dataset.
+- Troubleshoot run failures.
+- Manage personally identifiable information (PII).
+- Ensure compliance with data regulations.
+
+This guide provides information on which lineage data is automatically extracted on Astro and how to integrate external systems, including Databricks and dbt, that require additional configuration. To learn about how to view data lineage on Astro, see [View data lineage](data-lineage.md).
```
Comment (kwcanuck) [inline astro/set-up-data-lineage.md:17]:
```suggestion

```

File: astro/set-up-data-lineage.md [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
+
+## Data lineage on Astro
+
+To view data lineage from across your ecosystem, lineage data first needs to be extracted from external systems and stored in a lineage backend. Astro uses the [OpenLineage Airflow library](https://openlineage.io/integration/apache-airflow/) (`openlineage-airflow`) to extract lineage from Airflow tasks and stores that data in the Astro control plane. The latest version of the OpenLineage Airflow library is installed on [Astro Runtime](runtime-image-architecture.md) by default.
```
Comment (kwcanuck) [inline astro/set-up-data-lineage.md:21]: ``
To view data lineage data, it first needs to be extracted from an external application before it is stored in a lineage backend. Astro uses the [OpenLineage Airflow library](https://openlineage.io/integration/apache-airflow/) (`openlineage-airflow`) to extract lineage from Airflow tasks and stores that data in the Astro control plane. The latest version of the OpenLineage Airflow library is installed on [Astro Runtime](runtime-image-architecture.md) by default.
```

File: astro/set-up-data-lineage.md [hunk 4]
```diff
@@ -32,77 +40,42 @@ To emit lineage data from an Airflow task that runs outside of Astro or from an
 2. Copy the value in the **Lineage API Key** field.
 3. Specify your Organization's OpenLineage API key in the external system's configuration.
 
-## OpenLineage and Snowflake
+## Snowflake and OpenLineage with Airflow
 
-Lineage data emitted from [Snowflake](https://www.snowflake.com/en/) is similar to what is collected from other SQL databases, including Amazon Redshift and Google BigQuery. However, Snowflake is unique in that it emits [query tags](https://docs.snowflake.com/en/user-guide/object-tagging.html#what-is-a-tag) that provides additional task execution detail that other databases don't provide.
+Lineage data emitted from [Snowflake](https://www.snowflake.com/en/) is similar to what is collected from other SQL databases, including Amazon Redshift and Google BigQuery. However, Snowflake is unique in that it emits [query tags](https://docs.snowflake.com/en/user-guide/object-tagging.html#what-is-a-tag) that provide additional task execution detail that other databases don't provide.
```
Comment (kwcanuck) [inline astro/set-up-data-lineage.md:45]:
```suggestion
Lineage data emitted from [Snowflake](https://www.snowflake.com/en/) is similar to what is collected from other SQL databases, including Amazon Redshift and Google BigQuery. However, Snowflake is unique in that it emits [query tags](https://docs.snowflake.com/en/user-guide/object-tagging.html#what-is-a-tag) that provide additional task execution details.
```

File: astro/set-up-data-lineage.md [hunk 5]
```diff
@@ -32,77 +40,42 @@ To emit lineage data from an Airflow task that runs outside of Astro or from an
...
-
-Because Workspace permissions are not yet applied to the **Lineage** page, viewing source code for [supported Airflow operators](data-lineage-support-and-compatibility.md#supported-airflow-operators) is off by default. If you want users across Workspaces to be able to view source code for Airflow tasks in a given Deployment, create an [environment variable](environment-variables.md) in the Deployment with a key of `OPENLINEAGE_AIRFLOW_DISABLE_SOURCE_CODE` and a value of `False`. Astronomer recommends enabling this feature only for Deployments with non-sensitive code and workflows.
-
-
-## OpenLineage and Snowflake 
-
-Use the information provided here to set up lineage collection for Snowflake.
-
-### Setup
+To view this data, select a SnowflakeOperator task in the **Lineage** tab of the Cloud UI and click on the dataset. See [View data lineage](data-lineage.md#view-metrics-for-a-specific-run-or-dataset).
```
Comment (kwcanuck) [inline astro/set-up-data-lineage.md:65]:
```suggestion
To view this data, select a SnowflakeOperator task in the **Lineage** tab of the Cloud UI and click the dataset. See [View data lineage](data-lineage.md#view-metrics-for-a-specific-run-or-dataset).
```
image

File: astro/set-up-data-lineage.md [hunk 6]
```diff
@@ -32,77 +40,42 @@ To emit lineage data from an Airflow task that runs outside of Astro or from an
...
 
-Like other [supported Airflow operators](data-lineage-support-and-compatibility.md#supported-airflow-operators), the [SnowflakeOperator](https://registry.astronomer.io/providers/snowflake/modules/snowflakeoperator) has full data lineage support and capabilities by default. As long as you connect to Snowflake through an Airflow connection, any task using the SnowflakeOperator emits lineage data about the Snowflake tables it queries.
+:::tip
 
-### Verify 
+The ability to view source SQL code for [supported Airflow operators](data-lineage-support-and-compatibility.md#supported-airflow-operators) in the **Lineage** tab of the Cloud UI is off by default for all Workspace users. To enable source code, set the following [environment variable](environment-variables.md) for each Airfow Deployment:
```
Comment (kwcanuck) [inline astro/set-up-data-lineage.md:69]:
```suggestion
The ability to view source SQL code for [supported Airflow operators](data-lineage-support-and-compatibility.md#supported-airflow-operators) in the **Lineage** page of the Cloud UI is off by default for all Workspace users. To enable source code, set the following [environment variable](environment-variables.md) for each Airflow Deployment:
```
image

**Review by kwcanuck — APPROVED** [review]

File: astro/set-up-data-lineage.md (hunk 1 above)
Comment (paolaperaza) [reply astro/set-up-data-lineage.md:10]: Hm I played around with this Rob and I still think I like the logical order of:

- what is data lineage
- what can u get out of it
- what are you going to learn in this doc

File: astro/set-up-data-lineage.md [hunk 7]
```diff
@@ -32,77 +40,42 @@ To emit lineage data from an Airflow task that runs outside of Astro or from an
...
-Together, this information provides a robust amount of data to help data stakeholders analyze their pipelines and trace data.
-
-When collecting lineage data for Snowflake with Airflow emitting OpenLineage, you should expect to collect information about the input and output datasets of jobs for certain queries. This appears in the Lineage page in Astro as a link between the relevant Airflow task and the datasets it operates on. Data is also enriched by Snowflake’s query tags when running tasks on Airflow using the `SnowflakeOperator` and `SnowflakeAsyncOperator` by adding that tag to the job’s run facet.
-
-#### Setup
+- SQL queries. Only CREATE statements are currently supported.
+- Query duration. This is different from the Airflow task duration.
+- Input datasets
+- Output datasets
+- Quality metrics based on dataset and column-level checks, including successes and failures per run
```
Comment (paolaperaza) [inline astro/set-up-data-lineage.md:63]:
```suggestion
- SQL queries. Only CREATE statements are currently supported
- Query duration. This is different from the Airflow task duration
- Input datasets
- Output datasets
- Quality metrics based on dataset and column-level checks, including successes and failures per run
```

**Pull request merged**